pymc/model_graph.py

```diff
@@ -154,7 +169,7 @@ def _make_node(self, var_name, graph, *, nx=False, cluster=False, formatting: st
             shape = "box"
             style = "rounded, filled"
             label = f"{var_name}\n~\nMutableData"
-        elif v.owner and isinstance(v.owner.op, RandomVariable):
+        elif v.owner and isinstance(v.owner.op, (RandomVariable, SymbolicRandomVariable)):
```

Comment:
We should just check if a variable is in `model.basic_RVs` instead of forcing a specific type. Similarly, instead of checking if `observations` is in a tag to figure out if it is observed, we should check if it is in `model.observed_RVs`